src/demo/upload-demo.tsx

```
@@ -0,0 +1,165 @@
+    baseURL: '' as string | undefined,
+    timeout: 120000,
+    adapter: 'xhr',
+  })
+
+  async function uploadObjectDoc(presignedUrlData: any, uri: string, image: UploadFileType) {
+    try {
+      const {data: url} = presignedUrlData
+      const doc = await fetch(uri)
+      const blob = await doc.blob()
+      const awsAxios = axios.create({})
+      delete awsAxios.defaults.headers.common['Authorization']
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)